Wikipedia/Code/WMFCompassView.m

```diff
@@ -223,4 +223,21 @@ - (void)drawCompassLinesInContext:(CGContextRef)ctx
     }
 }
 
+#pragma mark - Accessibility
+
+- (BOOL)isAccessibilityElement {
+    return YES;
+}
+
+- (NSString *)accessibilityLabel {
+    double positive_angle = self.angleRadians >= 0 ? self.angleRadians : self.angleRadians + 2*3.14;
+    NSInteger clock_direction = lroundf(positive_angle / (3.14 / 6));
+    if (clock_direction == 0) {
+        clock_direction = 12;
+    }
+    NSString* label = MWLocalizedString(@"compass-direction", nil);
+    label = [label stringByReplacingOccurrencesOfString:@"$1" withString:[NSString localizedStringWithFormat:@"%@", @(clock_direction)]];
```

Comment:
i did a bit of research by turning on VoiceOver and using the built-in Compass app. It uses degrees for all languages I tested (English, Spanish, & Chinese), plus the initials the cardinal direction, e.g. 145 degrees, SE (it says "S-E" not "Southeast").  So I'd suggest:

- Converting radians to degrees (using `RADIANS_TO_DEGREES`)
- Optionally, adding cardinal direction (N, NE, E, SE, S, etc.)
- Localizing the label using the format: "$1 degrees" and, optionally, combining that w/ another localized string for the direction initials (i.e. `MWLocalizedString(@"west-initial", nil)` would be "W" in English, and "O" in Spanish)